ee/maintained-apps/ingesters/homebrew/ingester.go

```diff
@@ -142,7 +143,19 @@ func (i *brewIngester) ingestOne(ctx context.Context, app inputApp) (*maintained
 	}
 
 	out.Name = app.Name
-	out.Version = strings.Split(cask.Version, ",")[0]
+	if app.VersionRef != "" {
```

Comment:
Feels like it would make more sense to have a mutation function that operates on the entire app output right before it's returned, along the lines of the CVE mutation, and then inside that function either no-op or call other fns based on slug. Should avoid a bunch of complexity in this func, and if we need to override some other field (e.g. downloading a package to figure out its hash), we would need no further nodifications to `ingestOne`.